src/sagemaker/training_compiler/config.py

```diff
@@ -118,36 +118,25 @@ def _to_hyperparameter_dict(self):
     @classmethod
     def validate(
         cls,
-        image_uri,
-        instance_type,
-        distribution,
+        estimator,
```

Comment:
Is this backward compatible as you are removing args from it

Reply:
The validate function is not for direct use by customers. The estimators use it internally to validate customer's args.

To answer your question, this is not backwards compatible.

Reply:
Do you anticipate someone might have used it.

Reply:
Nope, we have limited customers. 
We have an established process for getting support through our PM.
I have a high level of confidence that nobody is using this function directly.